Basic/ClientDriven/Assets/Textures/backgroundGlow.png.meta

```diff
@@ -0,0 +1,146 @@
+fileFormatVersion: 2
+guid: 54479458116cb0b47af67b1eab91865a
+TextureImporter:
+  internalIDToNameTable: []
+  externalObjects: {}
+  serializedVersion: 11
+  mipmaps:
+    mipMapMode: 0
+    enableMipMap: 1
```

Comment:
As this is used in a screen-overlay UI mipmaps should be disabled (saves an amount of memory equal to ~33% of the texture's weight)